README.md

```diff
@@ -1,2 +1,58 @@
-# TDD--project
-This is a project to implement Test Driven Development
+![](https://img.shields.io/badge/Microverse-blueviolet)
+ # TDD--project
+ This is a project to implement Test Driven Development
+ 
+ ## Built With
+
+ - Ruby
+
+ - RSpec
+
+ ## Getting Started
+
+ To get a local copy up and running follow these simple example steps.
+- git clone  `https://github.com/richardoppiyo/TDD--project.git`
+
+- change to the directory using `cd TDD--project`
+
+- open with your preffered code editor.
+
+ ## Running the project
+
+To run the project,
+- Navigate to the root directory of the project `cd TDD--project`
+
+- run `rspec`
+
+-The test results will be displayed on the terminal.
+
+ ## Authors:
+
+### 👤 Joaquin G.L. Zanetti
+- GitHub: [@Kakalanp](https://github.com/Kakalanp)
+- Twitter: [@LeccaJoaquin](https://twitter.com/LeccaJoaquin)
+- LinkedIn: [Joaquín Garrido Lecca Zanetti](https://www.linkedin.com/in/joaquin-garrido-lecca-zanetti/)
+
+### 👤 **Richard Opiyo**
+
+- GitHub: [@richardoppiyo](https://github.com/richardoppiyo)
+- Twitter: [@blessed_ricky](https://twitter.com/blessed_ricky)
+- LinkedIn: [Richard Opiyo](https://linkedin.com/in/richardoppiyo) 
+
+ ## 🤝 Contributing
+
+ Contributions, issues, and feature requests are welcome!
+
+ Feel free to check the [issues page](https://github.com/usorfaitheloho/school-library/issues).
+
+ ## Show your support
+
+ Give a ⭐️ if you like this project!
+
+ ## Acknowledgments
+
+ - Microverse: [microverse community](https://github.com/microverseinc)
+
+ ## 📝 License
+
+ This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] I can see you added LICENSE to your project. However when I try to open it by Readme.md link it's not working as expected. Kindly connect your LICENSE file to your Readme.

Reply:
It is because of the license's name :sweat_smile:
Fixing it right now